sdks/python/build.gradle

```diff
@@ -35,7 +35,7 @@ task setupVirtualenv {
     }
     exec {
       executable 'sh'
```

Comment:
From the logs, `pip -V` output is:
> pip 10.0.1 from /usr/local/lib/python2.7/dist-packages/pip
when the path should be something like `${envdir}/bin/pip`.

It seem the virtualenv isn't being set up correctly. Perhaps changing `sh` to `bash` will fix that.